app/src/main/java/org/p2p/wallet/history/ui/token/TokenHistoryFragment.kt

```diff
@@ -170,6 +170,9 @@ class TokenHistoryFragment :
                 sellAnalytics.logTokenScreenActionClicked()
                 replaceFragment(SellPayloadFragment.create())
             }
+            else -> {
+                // TODO: WHAT SHOULD BE HERE?
+            }
```

Comment:
just insert Unit, it's totally ok

```suggestion
            else -> Unit
```